src/util.cpp

```diff
@@ -802,10 +802,11 @@ int GetNumCores()
 std::string CopyrightHolders(const std::string& strPrefix)
 {
     std::string strCopyrightHolders = strPrefix + _(COPYRIGHT_HOLDERS);
-    if (strCopyrightHolders.find("%s") != strCopyrightHolders.npos) {
+    if (strCopyrightHolders.find("%s") != std::string::npos) {
         strCopyrightHolders = strprintf(strCopyrightHolders, _(COPYRIGHT_HOLDERS_SUBSTITUTION));
     }
-    if (strCopyrightHolders.find("Bitcoin Core developers") == strCopyrightHolders.npos) {
+    if (std::string("Bitcoin Core") != COPYRIGHT_HOLDERS_SUBSTITUTION) {
```

Comment:
> Comparing two string literals will likely do what you want due to the compiler merging the constants to the same memory location. This is not behaviour to rely on though.

LOL yes 'it will mostly work' is the worst kind of working.

> if (strcmp("Bitcoin Core", COPYRIGHT_HOLDERS_SUBSTITUTION) != 0) can also be evaluated at compile time

I don't think we care here, it's not like this is a performance critical inner loop.